### DISCUSSION TITLE
内容管理标签重复显示

### DISCUSSION THREAD

**Discussion by qichunren** [geeknote-net/feedback#36] [General]
在个人内容管理的列表中，选一个文章点进去，再后退。这样多执行几次这样的操作，然后再进入具体的文章编辑界面，会看到标签输入控件重复渲染多行。

**Comment by chloerei** [comment]
是什么浏览器呢？方便的话能不能录个视频。

之前有这个问题，后来我在 mac + chrome 上修了， 可能其他浏览器不一样。

**Reply by qichunren** [reply]
[tag_issue.zip](https://github.com/geeknote-net/feedback/files/12448003/tag_issue.zip)
我的是Windows 10 专业版64位，Chrome Version 116.0.5845.111 (Official Build) (64-bit)
